src/scheduler/check.cpp

```diff
@@ -1578,8 +1578,8 @@ check_normal_node_path(status *policy, server_info *sinfo, queue_info *qinfo, re
 
 			/* Find the pset of the server which owns the job */
 			for (auto &spset : sinfo->svr_to_psets) {
-				if (spset.svr_inst_id == resresv->svr_inst_id) {
-					msvr_pset[0] = spset.np;
+				if (spset.first == resresv->svr_inst_id) {
+					msvr_pset[0] = spset.second;
```

Comment:
Same here, you just say  msvr_pset[0] = sinfo->svr_to_psets[resresv->svr_inst_id]